### DISCUSSION TITLE
curious about autogluon +gan

### DISCUSSION THREAD

**Discussion by hyunjun3** [autogluon/autogluon#3423] [General]
hello!
Is it possible that autogluon to generate time series data? such as RGAN

Thanks

**Comment by shchur** [comment]
Hello, we currently do not support the task of generating entire time series

**Comment by hyunjun3** [comment]
Thanks for giving me an information.



2023년 7월 26일 (수) 오후 8:23, Oleksandr Shchur ***@***.***>님이 작성:
